deployment/CE07SHSP_Deploy.csv

```diff
@@ -143,12 +143,12 @@ deploymentNumber,Reference Designator,sensor.uid,startDateTime,stopDateTime,lat,
 16,CE07SHSP-SP001-09-PARADJ000,CGINS-PARADJ-00411,2024-06-19T17:18:00,2024-07-18T21:42:00,46.98450,-124.55400,CGMCE-07SHSP-00016,STP20240618,CGVEH-CSPPCE-00003,1,,,,,87,87,,,
 16,CE07SHSP-SP001-06-SPKIRJ000,CGINS-SPKIRJ-00264,2024-06-19T17:18:00,2024-07-18T21:42:00,46.98450,-124.55400,CGMCE-07SHSP-00016,STP20240618,CGVEH-CSPPCE-00003,1,,,,,87,87,,,
 16,CE07SHSP-SP001-02-VELPTJ000,CGINS-VELPTJ-11779,2024-06-19T17:18:00,2024-07-18T21:42:00,46.98450,-124.55400,CGMCE-07SHSP-00016,STP20240618,CGVEH-CSPPCE-00003,1,,,,,87,87,,,
-17,CE07SHSP-SP001-00-SPPENG000,CGCON-CSPPCE-00006,2025-05-19T00:11:00,,46.98810,-124.56650,CGMCE-07SHSP-00017,SR2511,CGVEH-CSPPCE-00006,1,,,,,87,87,,,
-17,CE07SHSP-SP001-08-CTDPFJ000,CGINS-CTDPFJ-00405,2025-05-19T00:11:00,,46.98810,-124.56650,CGMCE-07SHSP-00017,SR2511,CGVEH-CSPPCE-00006,1,,,,,87,87,,,
-17,CE07SHSP-SP001-01-DOSTAJ000,CGINS-DOSTAJ-00353,2025-05-19T00:11:00,,46.98810,-124.56650,CGMCE-07SHSP-00017,SR2511,CGVEH-CSPPCE-00006,1,,,,,87,87,,,
-17,CE07SHSP-SP001-07-FLORTJ000,CGINS-FLORTJ-01206,2025-05-19T00:11:00,,46.98810,-124.56650,CGMCE-07SHSP-00017,SR2511,CGVEH-CSPPCE-00006,1,,,,,87,87,,,
-17,CE07SHSP-SP001-05-NUTNRJ000,CGINS-NUTNRJ-00871,2025-05-19T00:11:00,,46.98810,-124.56650,CGMCE-07SHSP-00017,SR2511,CGVEH-CSPPCE-00006,1,,,,,87,87,,,
-17,CE07SHSP-SP001-04-OPTAAJ000,CGINS-OPTAAJ-00283,2025-05-19T00:11:00,,46.98810,-124.56650,CGMCE-07SHSP-00017,SR2511,CGVEH-CSPPCE-00006,1,,,,,87,87,,,
-17,CE07SHSP-SP001-09-PARADJ000,CGINS-PARADJ-00505,2025-05-19T00:11:00,,46.98810,-124.56650,CGMCE-07SHSP-00017,SR2511,CGVEH-CSPPCE-00006,1,,,,,87,87,,,
-17,CE07SHSP-SP001-06-SPKIRJ000,CGINS-SPKIRJ-00252,2025-05-19T00:11:00,,46.98810,-124.56650,CGMCE-07SHSP-00017,SR2511,CGVEH-CSPPCE-00006,1,,,,,87,87,,,
-17,CE07SHSP-SP001-02-VELPTJ000,CGINS-VELPTJ-11900,2025-05-19T00:11:00,,46.98810,-124.56650,CGMCE-07SHSP-00017,SR2511,CGVEH-CSPPCE-00006,1,,,,,87,87,,,
+17,CE07SHSP-SP001-00-SPPENG000,CGCON-CSPPCE-00006,2025-05-20T00:11:00,,46.98810,-124.56650,CGMCE-07SHSP-00017,SR2511,CGVEH-CSPPCE-00006,1,,,,,87,87,,,
```

Comment:
Similar to Russ's comments about CE01, the longitude is not exactly the same as what is listed in the the daily report log on the 19th.  I calculate it should be  -124.56633 (from the report's (-124 - 33.980/60 = -124.56633) instead of -124.56650.
However, I don't really care if you change this or not.  I think -124.566 is good enough anyway, so it is only up to you if you want to change anything.

Reply:
Report includes the anchor survey results. Based on those (not the transom GPS), the deploy location of CE07SHSP is 46°59.283’N, 124°33.980’W (46.98805, -124.56633).

Reply:
The cruise report values that I see for ce07shsp are the bolded latlons for it in the Completed Section of the 5/19 cruise report and are 46° 59.283' / 124° 33.980'.  The activities log (at 19:20 local) in this cruise report stated that the ce07shsm and ce07shsp anchor survey points were reasonable, so I would assume that these are the anchor survey values.